cmd/otelcol/Dockerfile

```diff
@@ -31,5 +31,6 @@ COPY --from=smartagent /usr/lib/splunk-otel-collector/agent-bundle /usr/lib/splu
 COPY config/collector/gateway_config.yaml    /etc/otel/collector/gateway_config.yaml
 COPY config/collector/otlp_config_linux.yaml /etc/otel/collector/otlp_config_linux.yaml
 COPY config/collector/agent_config.yaml      /etc/otel/collector/agent_config.yaml
+COPY config/collector/fargate_config.yaml    /etc/otel/collector/fargate_config.yaml
 ENTRYPOINT ["/otelcol"]
-EXPOSE 13133 14250 14268 4317 6060 8888 9411 9443 9080
+EXPOSE 13133 14250 14268 4317 6060 8888 9411 9443 9080 6832 6831 55679
```

Comment:
Why are we exposing these? 68* are UDP ports -- are they needed for Fargate? 55679 should not be exposed by default

Reply:
If UDP ports are needed for sidecar mode then we should decide whether a separate Dockerfile should be used or not. In either case, you need `/udp` or this will not work (e.g. `6832/udp`)

Reply:
I removed ports 68* and 55679